sdk/security/keyvault/azsecrets/CHANGELOG.md

```diff
@@ -173,4 +173,4 @@
 * Adds the MIT License for redistribution
 
 ## 0.1.0 (2021-10-05)
-* This is the initial release of the `azsecrets` library
+* This is the initial release of the `azsecrets` library
```

Comment:
I guess this is a line ending change?

Reply:
Yeah, don't know why it's flagged as a change. Can't seem to fix it

Reply:
Looks like the change is deleting the newline. I don't know why GitHub doesn't add the usual "no terminal newline" glyph but we don't want to remove terminal newlines (or change line endings). The diff should disappear if you restore the newlines manually, or you can `git checkout origin/main -- *.md`

Reply:
That fixed it, thank you!